modules/Movecraft/src/main/java/net/countercraft/movecraft/async/translation/TranslationTask.java

```diff
@@ -440,7 +452,20 @@ private boolean checkFuel(){
         return true;
     }
 
-    private boolean inclineCraft(HashHitBox hitBox){
+    private MovecraftLocation surfaceLoc(MovecraftLocation ml) {
+        MovecraftLocation surfaceLoc = ml;
+        Material testType;
+        do {
+            surfaceLoc = surfaceLoc.translate(0, 1, 0);
+            if (surfaceLoc.getY() + 1 > craft.getType().getMaxHeightLimit()) {
```

Comment:
Why break in an if rather than as the while condition. This seems like it will stop working correctly at max hight